.rubocop.yml

```diff
@@ -0,0 +1,17 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
@jeffse did you mean to add these onto your local master instead of the branch?

We need to get this PR into a state where it only has your commits on it - let me know if I can help with that at all.